Doc/Tutorial/chapter_contributing.tex

```diff
@@ -66,15 +65,18 @@ \section{Maintaining a distribution for a platform}
 \end{verbatim}
 
 This will create an RPM for your specific platform and a source RPM in the directory \verb|dist|. This RPM should be good and ready to go, so this is all you need to do! Nice and easy.
-
+ 
```

Comment:
Have you accidentally added trailing white space here?

Reply:
yes